src/app/shared/workitem-utils.ts

```diff
@@ -11,6 +11,36 @@ export function filterOutClosedItems(items) {
   });
 }
 
+export function buildAssigneeQuery(filterService: FilterService, userId) {
+  return filterService.queryJoiner(
+    {},
+    filterService.and_notation,
+    filterService.queryBuilder(
+      'assignee', filterService.equal_notation, userId
+    )
+  );
+}
+
+export function buildSpaceQuery(filterService: FilterService, spaceId) {
+  return filterService.queryBuilder(
+    'space', filterService.equal_notation, spaceId
+  );
+}
+
+export function buildClosedWorkItemQuery(filterService: FilterService) {
```

Comment:
missing type definition for the function, whatever type `stateQuery` returns

Reply:
@aptmac `queryJoiner` and `queryBuilder` doesn't return any specific type of object